.github/workflows/bump.yml

```diff
@@ -30,6 +30,13 @@ jobs:
   cargo-upgrades:
     runs-on: "lab"
     steps:
+      # Use a GitHub App token so that the generated PR can trigger CI
+      - name: "Generate GitHub App token"
+        id: "app-token"
+        uses: "actions/create-github-app-token@v2"
```

Comment:
Does creating this token increase permissions beyond triggering CI?  What roles does this token give the CI job?

Reply:
Looks like permission are restricted to what the github app can do so this is fine.